routers/api/v1/repo/issue.go

```diff
@@ -963,3 +963,534 @@ func UpdateIssueDeadline(ctx *context.APIContext) {
 
 	ctx.JSON(http.StatusCreated, api.IssueDeadline{Deadline: &deadline})
 }
+
```

Comment:
It's better to move all these new router functions to a new file. Maybe we could name it `issue_dep.go` because the current file is too big. But I think this is not a block. We can also do that next PR.